certified-connectors/Taqnyat SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,393 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Taqnyat",
+    "description": "Taqnyat integration for short messages \"SMS\"",
+    "version": "1.0",
+    "contact": {
+      "name": "Taqnyat Support",
+      "url": "https://www.taqnyat.sa/",
+      "email": "support@taqnyat.sa"
+    },
+    "license": {
+      "name": "MIT for OAS 2.0",
+      "url": "https://opensource.org/licenses/MIT"
+    }
+  },
+  "host": "api.taqnyat.sa",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.taqnyat.sa/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.taqnyat.sa/en/privacy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Communication;Website"
+    }
+  ],
+  "paths": {
+    "/v1/messages": {
+      "post": {
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "Response to successful send sms.",
+            "x-ms-summary": "Success response",
+            "schema": {
+              "$ref": "#/definitions/SendSMSSuccessResponseBody"
+            }
+          },
+          "400": {
+            "description": "Response to failed send sms.",
+            "x-ms-summary": "Failed response",
+            "schema": {
+              "$ref": "#/definitions/FailureResponseBody"
+            }
+          },
+          "default": {
+            "description": "Response to failed send sms.",
+            "x-ms-summary": "Failure response",
+            "schema": {
```

Comment:
@TaqnyatNocTeam The 'default' response should not have schema definition. Schemas should be defined on expected responses only. Please remove the schema.